charts/cloudzero-agent/values.yaml

```diff
@@ -47,7 +47,7 @@ prometheusConfig:
     # -- Any items added to this list will be added to the Prometheus scrape configuration.
     additionalScrapeJobs: []
 
-kube-state-metrics:
+kube_state_metrics:
```

Comment:
The `-` were actually causing issues in the ConfigMap file, so I switched to `_`, which seemed to fix it.

Reply:
It's been a little while, but I believe you can use this form:
```
value: {{ index .Values "some-key" "nested-value" }}
```

Not sure but maybe this also:
```
value: {{ .Values."some-key"."nested-value" }}
```